docs/tutorials/web/excel-addin-bert-js.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Custom Excel Functions for BERT Tasks in JavaScript
+description:  Custom Excel Functions for BERT Tasks in JavaScript
+parent: Deploy on web
+grand_parent: Tutorials
+has_children: false
+nav_order: 2
+---
+
+# ONNX Runtime Custom Excel Functions for BERT NLP Tasks in JavaScript
+{: .no_toc }
+
+In this tutorial we will look at how we can create custom excel functions (`ORT.Sentiment()` and `ORT.Question()`) to implement BERT NLP models with ONNX Runtime Web to enable deep learning in spreadsheet tasks. The inference happen locally in the browser with excel on the web. 
+
+<div class="embed-container">
+  <iframe
+      src="https://www.youtube.com/embed/wuSxWGY_Sjg"
+      width="560" height="315"
+      frameborder="0"
+      allowfullscreen="true">
+  </iframe>
+</div>
+
+## Contents
+{: .no_toc }
+
+* TOC placeholder
+{:toc}
+
+## Prerequisites
+
+- [See the prerequisites from this tutorial](https://learn.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial-create-custom-functions?source=recommendations&tabs=excel-windows#prerequisites)
+- [ONNX Runtime Web](https://onnxruntime.ai/docs/get-started/with-javascript.html#:~:text=ONNX%20Runtime%20Web%20You%20can%20install%20and%20import,syntax%20%28recommended%29%20import%20%2A%20as%20ort%20from%20%27onnxruntime-web%27%3B)
+- This plugin currently doesn't work in the client version of excel.
```

Comment:
Super nit: this isn't really a pre-requisite, more of a limitation

Reply:
removed!